.github/workflows/main.yml

```diff
@@ -46,52 +42,54 @@ jobs:
           BRANCH_NAME=`echo $BRANCH_NAME | sed "s/[^[:alnum:]-]//g" | tr '[:upper:]' '[:lower:]'` # lowercase, only alphanumeric and dash
           echo ::set-env name=BRANCH_NAME::$(echo $BRANCH_NAME)
 
-          if [[ "${{ github.event_name}}" == "push" && "$BRANCH_NAME" == "master" ]]
-          then
-            echo "Running push on master branch. Release will happen if version was bumped."
+          FULL_REPO_SLUG=`echo "${{ github.event.pull_request.head.repo.full_name }}" | sed "s/[^[:alnum:]-]//g" | tr '[:upper:]' '[:lower:]'` # lowercase, only alphanumeric and dash
+          echo ::set-env name=FULL_REPO_SLUG::$(echo $FULL_REPO_SLUG)
 
-            REPO_VERSION_HIGHEST=$(git describe --tags --always --abbrev=0)
-            echo ::set-env name=REPO_VERSION_HIGHEST::$(echo $REPO_VERSION_HIGHEST)
-            echo "Highest released version was $REPO_VERSION_HIGHEST"
+          REPO_VERSION_HIGHEST=$(git describe --tags --always --abbrev=0)
+          echo "Highest released version was: $REPO_VERSION_HIGHEST"
 
-            REPO_VERSION_DESIRED=$(cargo pkgid --manifest-path libvcx/Cargo.toml | cut -d# -f2 | cut -d: -f2)
-            echo ::set-env name=REPO_VERSION_DESIRED::$(echo $REPO_VERSION_DESIRED)
-            echo "Repo declares version $REPO_VERSION_DESIRED"
+          if [[ "${{ github.event_name }}" == "pull_request" ]]
+          then
+            REPO_HEAD="${{ github.event.pull_request.head.repo.full_name }}" # source repository
+            REPO_BASE="${{ github.event.pull_request.head.base.full_name }}" # target repository
+            echo "This is pull request from $REPO_HEAD to $REPO_BASE"
 
-            if [[ "$REPO_VERSION_HIGHEST" == "$REPO_VERSION_DESIRED" ]]
+            if [[ "$REPO_HEAD" == "AbsaOSS/libvcx" ]]
```

Comment:
Can this be made [more generic](https://github.community/t/how-to-detect-a-pull-request-from-a-fork/18363/3)?